src/applications/check-in/pre-check-in/hooks/useFormRouting.jsx

```diff
@@ -17,6 +17,44 @@ const useFormRouting = (router = {}) => {
     [dispatch],
   );
 
+  const goToErrorPage = useCallback(
+    () => {
+      dispatchGoToNextPage(URLS.ERROR);
+      router.push(URLS.ERROR);
+    },
+    [dispatchGoToNextPage, router],
+  );
+
+  const jumpToPage = useCallback(
+    (page, options = {}) => {
+      if (Object.values(URLS).includes(page)) {
+        const nextPage = page;
+        dispatchGoToNextPage(nextPage);
+        // check for params
+        const query = {
+          pathname: nextPage,
+        };
+        const { params } = options;
+        if (params) {
+          // get all url keys
+          const keys = Object.keys(params.url);
+          const queryParams = keys
+            .map(key => `${key}=${params.url[key]}`)
+            .join('&');
```

Comment:
```suggestion
          const queryParams = new URLSearchParams(params.url).toString();
```

! I haven't tested this !

It would also need `import URLSearchParams from 'url-search-params';`

Reply:
Added and tested your suggestion!